### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
Solves the following WCA Forum posts:

- [Typo in regulation 10e3](https://forum.worldcubeassociation.org/t/typo-in-regulation-10e3/14126).
- [Typo in 9b4a](https://forum.worldcubeassociation.org/t/typo-in-9b4a/14440).
- [Wording of Guideline 1h1+)](https://forum.worldcubeassociation.org/t/wording-of-guideline-1h1/14469).

### CODE REVIEW

**Comment by xsrvmy** [comment]
I think "the" is added in the wrong place in 2a3.

File: wca-regulations.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 2a) Any person may compete in a WCA competition if they (complemented by [Regulation Y3](regulations:regulation:Y3)):
     - 2a1) Comply with WCA Regulations.
     - 2a2) Meet the competition requirements, which must be clearly announced before the competition.
-    - 2a3) Are not suspended by WCA Board.
+    - 2a3) Are not suspended the by WCA Board.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
    - 2a3) Are not suspended by the WCA Board.
```

File: wca-regulations.md [hunk 2]
```diff
@@ -279,7 +279,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 10e) A puzzle is solved when all colored parts are reassembled and all parts are aligned within the limits specified below:
     - 10e1) For every two adjacent parts (e.g. two parallel, adjacent slices of a cube) of the puzzle that are misaligned by more than the limit described in [Regulation 10f](regulations:regulation:10f), the puzzle is considered to require one additional move to solve (see "Outer Block Turn Metric" in [Article 12](regulations:article:12)).
     - 10e2) If no further moves are required to bring the puzzle to its solved state, the puzzle is considered solved without penalty.
-    - 10e3) If one move is required, the puzzle is be considered solved with a time penalty (+2 seconds).
+    - 10e3) If one move is required, the puzzle is considered solved with a time penalty (+2 seconds).
```
Comment (Ivan-Brigidano) [inline wca-regulations.md:282]:
```suggestion
    - 10e3) If one move is required, the puzzle is to be considered solved with a time penalty (+2 seconds).
```

File: wca-regulations.md [hunk 3]
```diff
@@ -279,7 +279,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 10e) A puzzle is solved when all colored parts are reassembled and all parts are aligned within the limits specified below:
     - 10e1) For every two adjacent parts (e.g. two parallel, adjacent slices of a cube) of the puzzle that are misaligned by more than the limit described in [Regulation 10f](regulations:regulation:10f), the puzzle is considered to require one additional move to solve (see "Outer Block Turn Metric" in [Article 12](regulations:article:12)).
     - 10e2) If no further moves are required to bring the puzzle to its solved state, the puzzle is considered solved without penalty.
-    - 10e3) If one move is required, the puzzle is be considered solved with a time penalty (+2 seconds).
+    - 10e3) If one move is required, the puzzle is to be considered solved with a time penalty (+2 seconds).
```
Comment (lgarron) [inline wca-regulations.md]: I would instead remove a word to match the adjacent Regs.
```suggestion
    - 10e3) If one move is required, the puzzle is considered solved with a time penalty (+2 seconds).
```

**Review by lgarron — APPROVED** [review]
Thanks!

**Pull request merged**